tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py

```diff
@@ -933,3 +933,69 @@ def process(self, raw, identifiers, frame_meta):
             results.append(DetectionPrediction(identifier, labels, scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return results
+
+
+class SSDVGG16Adapter(Adapter):
+    __provider__ = 'ssd_vgg16'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'scores_out': StringField(description='scores output'),
+            'boxes_out': StringField(description='boxes output')
+        })
+        return params
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+        self.confidence_threshold = 0.01
+        self.iou_threshold = 0.45
+        self.keep_top_k = 200
```

Comment:
please provide opportunity to set these parameters via config

Reply:
Done